Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN adduser -u 9000 -D app
 COPY Gemfile Gemfile.lock /usr/src/app/
 
 RUN apk add --update build-base git && \
-    gem install bundler && \
+    gem install bundler -v 2.0.2 && \
```

Comment:
Matches up https://github.com/codeclimate/codeclimate-rubocop/blob/channel/rubocop-0-76/Gemfile.lock#L96
We might want to eventually update the latest, but it will require some more digging to figure out what needs to be adjusted.